update

```diff
@@ -337,6 +360,7 @@ const displayStats = (letsgo, data, o, s) => {
   }
   fs.writeFileSync('data/index.json', stringify(index.options));
   fs.writeFileSync('data/stats/index.json', stringify(index.stats));
+  fs.writeFileSync('data/full/index.json', stringify(index.stats));
```

Comment:
I assume this is what you meant @scheibo

Reply:
hmm, but i think this will kind of clobber `index.stats`, no? like `index.full` instead of repurposing `index.stats` to track the sizes of just the files in the `full/` folder. right now afaict the line above this one will actually be writing the sizes for the full folder into the stats folder because line 351 overwrites 350?

Reply:
Yep, that's what I get for copy-pasting without thinking.

Modified to add `full` attribute and then write it at the end.

https://github.com/pkmn/randbats/pull/13/commits/c2cdb311d62820f48d257291e55bdb23f8ef4ad2